esf_main.tex

```diff
@@ -875,11 +875,11 @@ \subsubsection{Charging}
 		\label{Elcon Battery Charger}
 	\end{figure}
 
-	The Charging system consists of two main components:  A Charger that connects to the main pack connector and supplies the current needed to charge the accumulator, and a shutdown and monitoring circuit that provides power to the AMS, opens the AIRS if the shutdown button is pressed, and displays critical information while charging. 
+	The charging system consists of two main components:  A charger that connects to the main pack connector and supplies the current needed to charge the accumulator, and a charging circuit that provides power to the AMS, opens the AIRS if the shutdown button is pressed, and displays critical information while charging. 
 	
-	This CE-Listed charger supplies constant current at 6A until cell voltages reach 4.15V. Then, constant voltage is supplied at 4.15V per cell, summing to 298.8 volts, until the cells are fully charged. The charging power lines ???TODO: Talk to Alex about fusing???. 
+	This CE-Listed charger supplies constant current at 6A until cell voltages reach 4.15V. Then, constant voltage is supplied at 4.15V per cell, summing to 298.8 volts, until the cells are fully charged. The charging power lines are protected via the charger's short-circuit protection, which automatically stops charging if short-circuit is encountered across the output.
 	
-	The charger interfaces with the accumulator through a lever action locking connector from TE Connectivity's HVP 800 series (P/Ns 2141227(accumulator side) and 2141154(charger side). These connectors contain an interlock that opens the AIRS when not engaged to maintain compliance with EV8.3.3. The charger has an interlock function that stops charging when the connector is not engaged, shown in figure \fref{Charger Interlock}. 
+	The charger interfaces with the accumulator through a lever action locking connector from TE Connectivity's HVP 800 series (P/Ns 2141227(accumulator side) and 2141154(charger side)). These connectors contain an interlock that opens the AIRS when not engaged to maintain compliance with EV8.3.3. The charger has an interlock function that stops charging when the connector is not engaged, shown in figure \fref{Charger Interlock}. 
```

Comment:
'Opens the shutdown circuit, preventing the AIRs from closing' is the wording I would use. 

Also your `\fref{}` won't work so well if you don't use the fig: prefix in your label like `\label{fig:whatever}`